assessments/unit-01/code.js

```diff
@@ -5,3 +5,107 @@ const emit = (a, b, product) => {
 };
 
 // Write your code here ...
+const averageWeight = (totalWeight, numItems) => {
+  return (totalWeight/numItems)
+}
+
+const hypotenuse = (a, b) => {
+  return Math.sqrt(a**2 + b**2)
+}
+
+const maxRadius = (widthOfArea, heightOfArea) => {
+  return (Math.min (widthOfArea, heightOfArea))/2
```

Comment:
This is correct but formatted weirdly. I'd change it like this:

```suggestion
  return Math.min(widthOfArea, heightOfArea) / 2
```